00-Install_and_Setup/requirements.txt

```diff
@@ -19,11 +20,11 @@ jupyterlab
 jupyter-offlinenotebook
 nbzip
 nbgitpuller
-astropy>=6.0
+astropy>6.0
```

Comment:
Because 6.0 has had bug fixes so it seemed like the proper pin was 6.0.x or higher. That may or may not be what my change does since I'm not super-familiar with how pip treats pines

Reply:
I think `=` would automatically pick up latest bugfix from 6.0.x series. That said, 6.0.x is not compatible with numpy 2.0, so maybe you really want `>=6.1`.